packages/components/src/components/hds/card/container.ts

```diff
@@ -5,31 +5,45 @@
 
 import Component from '@glimmer/component';
 import { assert } from '@ember/debug';
+import { BackgroundValues, LevelValues, OverflowValues } from './types.ts'
+import type { Background, Level, Overflow } from './types.ts';
```

Comment:
we're settling on prefixing these variables (these changes would unfortunately need to cascade across the files)
```suggestion
import { HdsCardBackgroundValues, HdsCardLevelValues, HdsCardOverflowValues } from './types.ts'
import type { HdsCardBackgrounds, HdsCardLevels, HdsCardOverflows } from './types.ts';
```